data/korath/korath variants.txt

```diff
@@ -678,6 +682,12 @@ ship "Ra'gru Ik 618" "Ra'gru Ik 618 (Grab)"
 	turret "Banisher Grav-Turret"
 
 
+# This variant needs to have the same load-out as the default 618 to
+# keep the intended difficulty level of the Korath Artifacts 2  mission
```

Comment:
Why is this comment necessary? This variant is only used in this mission, so my first impression is that this label is here to ensure that no-one changes the configuration of the "Ground Assault" variant in the future. If this problem is likely, then I would suggest using a name that isn't directly tied to a specific role that this ship doesn't actually look like it is setup for.

Reply:
It is now "Sentry" as suggested in your other comments.